test/dialect/mssql/test_compiler.py

```diff
@@ -1195,15 +1195,15 @@ def test_try_cast(self):
         )
 
     @testing.combinations(
-        ("no_persisted", "", ...),
+        ("no_persisted", "", Ellipsis),
```

Comment:
I had never heard of this object before.  what does it mean in this context?

Reply:
Is is used sometimes in the indexes, like numpy uses it sometimes. It is not very used otherwise.

Here it is just a flag that I use to check it `persisted` should be added to the attributes.
I guess `skip_arg` or a `flag=object()` would also work here

Reply:
I guess it has also been repurposed a bit in the typing library to generate empty functions/classes and specify some types

Reply:
**mike bayer** (zzzeek) wrote:

python 2 is not accepting this.